### PR TITLE
Feature/8-create-presentation-layer-with-draft-api-contoller

### PR DESCRIPTION
**Pull request by JmlKhan**
Created Controllers folder and added OfficeController to it. 
Installed and set up swagger.

### CODE REVIEW

**Comment by JmlKhan** [comment]
@apilatau

File: TelegramBotAPI/Controllers/OfficeController.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+
+namespace PresentationLayer.Controllers
+{
+    [Route("api/[controller]")]
+    
+    public class OfficeController : ControllerBase
```
Comment (apilatau) [inline TelegramBotAPI/Controllers/OfficeController.cs]: Add ApiController attribute and instead of Controller base , use Controller as a parent

**Review by apilatau — CHANGES_REQUESTED** [review]

File: TelegramBotAPI/Controllers/OfficeController.cs (hunk 1 above)
Comment (JmlKhan) [reply TelegramBotAPI/Controllers/OfficeController.cs]: Done

**Comment by JmlKhan** [comment]
Fixed merge conflict @apilatau

**Pull request merged**